### ISSUE TITLE
Identifier '_createElementVNode' has already been declared after install

### ISSUE THREAD

**Issue by AlexanderBredun** [storybook-vue/storybook-nuxt#76]
Help me please i get error after minimal setup
![image](https://github.com/storybook-vue/storybook-nuxt/assets/31702523/1c0c6504-f14a-4064-ab20-ff60717346d2)

i tried it with nuxt 3.7 and 3.6 i get this error everywhere 
i have this problem in this repo: https://github.com/AlexanderBredun/nuxt3-storybook
node: v18.16
npm: v9.5.1

**Comment by chakAs3** [comment]
please check https://storybook.nuxtjs.org/examples/basic

**Comment by bmulholland** [comment]
See also:
* https://github.com/storybook-vue/storybook-nuxt/issues/75
* https://github.com/nuxt-modules/storybook/issues/449
* https://github.com/nuxt/nuxt/issues/23363

**Comment by AlexanderBredun** [comment]
> See also:
> 
> * [Compatibility issues with Nuxt 3.7.x #75](https://github.com/storybook-vue/storybook-nuxt/issues/75)
> * [Storybook crashing with `The symbol "_withScopeId" has already been declared` error nuxt-modules/storybook#449](https://github.com/nuxt-modules/storybook/issues/449)
> * [Bug:SyntaxError: Identifier '_createElementVNode' has already been declared nuxt/nuxt#23363](https://github.com/nuxt/nuxt/issues/23363)

What exactly i needed to see there?) Like this module doesnt work with 3.7 nuxt? But i tried 3.6 and it still doesnt work. At this point, I am looking for a setup that works with any version of Nuxt and Storybook. Do you have one?

**Comment by bmulholland** [comment]
No, I am looking for the same and cannot get it. The latest is https://github.com/nuxt-modules/storybook/issues/428#issuecomment-1794448129
